Model/Layer/Filter/Stock.php

```diff
@@ -75,6 +76,12 @@ public function apply(RequestInterface $request)
             $this->_createItem($this->getLabel($filter), $filter)
         );
 
+        if ($request->getParam($this->getUrlParamName(), null) > 0
+            && $this->isSearchEngineElasticsearch() && $this->isEnabledInStockFilterOnElasticSide()
```

Comment:
if you putting conditions to next line, then to be consistent and increase readability - start all conditions from new line

Reply:
done